pages/terms-of-service.tsx

```diff
@@ -115,8 +116,8 @@ const TermsOfService = (props: Props) => {
           </p>
           <p>
             If you know or suspect that anyone other than you knows your user information (such as
-            an identification code or user name) and/or password you must promptly notify us at
-            contact@ferdium.org.
+            an identification code or user name) and/or password you must promptly{" "}
+            <Link href='/contact'>contact</Link> us.
```

Comment:
For security issues, I think an email is more suitable than a public GitHub issue or a message on Discord. Should we add the email in the contact page ?

Reply:
I think adding an email would be great as it also doesn't require signup to an external service. Who would be the right person to set that up?  Also, we might want to add it after setting it up, so we don't void any incoming emails?

Reply:
contact@ferdium.org already exists :-) So you can use this one.

Reply:
Perfect, on it!

Reply:
Added in f4b604e